manifests/t/t0rre/SoundPrank/1.0.1/t0rre.SoundPrank.yaml

```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: t0rre.SoundPrank
+PackageVersion: 1.0.1

```

Comment:
```suggestion
PackageVersion: 1.0.0
```

Reply:
Unfortunately, this must match Control Panel because WinGet relies on the DisplayVersion registry key.

Since `1.0.1` is bigger than `1.0.0` from the registry, this application would reinstall on a user's PC whenever they do `winget upgrade`.